src/utils.rs

```diff
@@ -6,6 +6,33 @@ use anyhow::Result;
 use reqwest::blocking::Request;
 use url::{Host, Url};
 
+pub fn unescape(text: &str, special_chars: &'static str) -> String {
```

Comment:
It took me a minute to figure out that e.g. `--form '\]=a'` is why `special_chars` can have different values. Can you add a test case?